app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -18,8 +17,12 @@ def initialize
         )
       end
 
-      def extract_token_from_credentials(credentials)
-        @extract_token_from_credentials.call(credentials: credentials)
+      def create_authentication_parameters(authenticator_input)
+        @authentication_parameters = Authentication::AuthnJwt::AuthenticationParameters.new(authenticator_input)
+        @authentication_parameters.credentials = authenticator_input.request.read
```

Comment:
can this be done in the `AuthenticationParameters` init method?

Reply:
Let's say that I would like to not couple `AuthenticationParameters` to request object
But I'll remove unnecessary credentials assignment.